config/redirects.yml

```diff
@@ -391,9 +391,9 @@ redirects:
   "/blog/tag/returning-to-teaching": "/returning-to-teaching"
 
 # Subject pages
-  "/subjects/physics": "/life-as-a-teacher/physics"
-  "/subjects/maths": "/life-as-a-teacher/maths"
-  "/subjects/engineers-teach-physics": "/life-as-a-teacher/engineers-teach-physics"
+  "/subjects/physics": "/life-as-a-teacher/what-you-could-teach/physics"
```

Comment:
These three changes all look fine but I'd suggest moving them under the `# Life as a teacher new section` part of this redirect file (and deleting `# Subject pages` just to make future maintenance a bit easier?